src/components/task-step/step-mixin.cjsx

```diff
@@ -5,10 +5,13 @@ BS = require 'react-bootstrap'
 {TaskStepActions, TaskStepStore} = require '../../flux/task-step'
 LoadableItem = require '../loadable-item'
 {CardBody} = require '../pinned-header-footer-card/sections'
-
+_ = require 'underscore'
 
 module.exports =
 
+  getInitialState: ->
+    classNames: ['task-step']
```

Comment:
Why store the whole array of `classNames` in the state instead of just `hasBanner`?